src/common/services/session/session.service.spec.js

```diff
@@ -302,6 +303,35 @@ describe('session service', function () {
       })
     });
 
+    it('should trigger giveSignInSuccess event', done => {
+      $httpBackend.expectPOST('https://give-stage2.cru.org/okta/login').respond(200, 'success')
+
+      jest.spyOn(sessionService.authClient, 'isAuthenticated').mockReturnValueOnce($q.resolve(true))
+      jest.spyOn(sessionService.authClient.tokenManager, 'getTokens').mockReturnValueOnce($q.resolve({
+        accessToken: {
+          accessToken: 'accessToken'
+        }
+      }))
+
+      const $injector = {}
+      jest.spyOn(angular, 'injector').mockReturnValue($injector)
+
+      const giveSignInSuccessCallback = jest.fn()
+      window.document.body.addEventListener('giveSignInSuccess', giveSignInSuccessCallback, { once: true })
```

Comment:
Is it best to make `giveSignInSuccess` a variable since we use it in a few places?

Reply:
You're right, we no longer use it, so it might be useful to remove `LoginOktaOnlyEvent` in a later PR

Reply:
#1203